src/calendarevent.h

```diff
@@ -205,10 +197,35 @@ private slots:
     void rsvpChanged();
     void externalInvitationChanged();
 
+protected:
+    CalendarData::Event *mData;
```

Comment:
Any reason to have this dynamically allocated or could it be without the pointer part?

Reply:
It's the same reason here. I don't like it neither, but I had no idea how to easily circumvent, without creating an addition file for the enum definitions.